sale_order_amount_to_invoice/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    amount_to_invoice = fields.Monetary(
+        compute="_compute_amount_to_invoice", store=True
+    )
+
+    @api.depends(
+        "order_line.untaxed_amount_invoiced", "order_line.untaxed_amount_to_invoice"
```

Comment:
Question: why do we need `order_line.untaxed_amount_invoiced` in dependency if it's not used in calculation?

Reply:
Thanks, I'll remove it)